book/tour/external-types.html

```diff
@@ -155,14 +155,14 @@ <h1 id="external-type"><a class="header" href="#external-type">External type</a>
 <code>queue</code> module to work with the new <code>Queue</code> type.</p>
```

Comment:
I ran `make` with mdbook, but my guess is that the version I got is different and made a lot of other changes.

I only accepted the obvious changes.